src/server_manager/ui_components/app-root.html

```diff
@@ -42,8 +43,20 @@
     <style>
       :host {
         width: 100%;
-        --primary-green: #00bfa5;
-        --light-gray: color: rgba(0,0,0,0.54);
+        --status-bar-width: 48px;
+      }
+      .app-container {
+        margin: 0 auto;
+      }
+      @media (min-width: 1281px) {
```

Comment:
There's an implied magic figure of 80px in these `@media` declarations - how did you arrive at that figure?

Reply:
These are the "standard" resolutions for tablet, laptop, and desktop. I have added comments.
(See these for [more](https://gist.github.com/gokulkrishh/242e68d1ee94ad05f488) [information](https://css-tricks.com/snippets/css/media-queries-for-standard-devices/)).